plugin/src/main/java/org/opensearch/ml/helper/ConnectorAccessControlHelper.java

```diff
@@ -70,13 +83,51 @@ public void validateConnectorAccess(Client client, String connectorId, ActionLis
             getConnector(client, connectorId, ActionListener.wrap(connector -> {
                 boolean hasPermission = hasPermission(user, connector);
                 wrappedListener.onResponse(hasPermission);
-            }, e -> { wrappedListener.onFailure(e); }));
+            }, wrappedListener::onFailure));
         } catch (Exception e) {
             log.error("Failed to validate Access for connector:" + connectorId, e);
             listener.onFailure(e);
         }
     }
 
+    public void validateConnectorAccess(
```

Comment:
Will the above [method](https://github.com/opensearch-project/ml-commons/pull/3382/files#diff-dd3a1f0ea8e2b72fe2f02728331c4e961f63c79ea4b4424410feb8abab04f37cR75) be removed once all APIs are refactored?

Reply:
Yes